react/features/filmstrip/actions.web.js

```diff
@@ -40,37 +49,55 @@ export * from './actions.any';
  * resolved to Redux state using the {@code toState} function.
  * @returns {Function}
  */
-export function setTileViewDimensions(dimensions: Object) {
+export function setTileViewDimensions() {
     return (dispatch: Dispatch<any>, getState: Function) => {
         const state = getState();
         const { clientHeight, clientWidth } = state['features/base/responsive-ui'];
-        const { disableResponsiveTiles, disableTileEnlargement } = state['features/base/config'];
         const {
-            height,
-            width
-        } = calculateThumbnailSizeForTileView({
-            ...dimensions,
-            clientWidth,
-            clientHeight,
             disableResponsiveTiles,
-            disableTileEnlargement
-        });
-        const { columns, rows } = dimensions;
+            disableTileEnlargement,
+            tileView = {}
```

Comment:
What's this config about?

Reply:
I've added it with a comment in config.js.

Basically our desired number of visible participants before was `maxColumns^2` and now this is replaced by this config option.